wiki/RTL-Guidelines.md

```diff
@@ -19,6 +20,15 @@ In general, the passage of time is depicted as left to right for LTR languages,
 
 When a UI is changed from LTR to RTL (or vice-versa), it’s often called mirroring. An RTL layout is the mirror image of an LTR layout, and it affects layout, text, and graphics.
 
+# How to enable RTL?
```

Comment:
There is a second option of setting the phone language to an RTL lsngusge like Arabic. What  do you think about adding that?

Reply:
DONE.